app/assets/javascripts/shortlink.js

```diff
@@ -0,0 +1,27 @@
+!function () {
+  function bindShortlink(form) {
+    var form = $(form)
+      , linkField = form.find('.shortlink-field')
```

Comment:
Comma warnings can be turned off with 'laxcomma'.<br>Bad line breaking before ','.